lms/static/sass/shared/_course_object.scss

```diff
@@ -180,78 +191,116 @@
         }
       // }
 
-      .info {
-        background: $content-wrapper-bg;
-        height: 220px + 130px;
-        left: 0px;
-        position: absolute;
-        top: 0px;
-        @include transition(all 0.15s linear 0s);
-        width: 100%;
-        overflow: hidden;
-
+      .course-image {
         .cover-image {
-          height: 200px;
+          height: 160px;
           overflow: hidden;
-          width: 100%;
+
+          &:before {
+            @include fade-transition();
+            content: '';
+            position: absolute;
+            top: 0;
+            left: 0;
+            background: black;
+            opacity: 0;
+            width: 100%;
+            height: 160px;
+            z-index: 10;
+          }
+
+          &:after {
+            @include fade-transition();
+            @include box-sizing(border-box);
+            @include calc( top, '80px - 25px');
+            @include calc( left, '50% - 100px');
+            content: 'LEARN MORE';
+            position: absolute;
+            width: 200px;
+            height: 50px;
+            z-index: 20;
+            border: 3px solid white;
+            border-radius: 3px;
+            font-family: $sans-serif;
+            font-weight: $font-semibold;
+            font-size: 1.1em;
+            line-height: 44px;
```

Comment:
If you need a specific line-height, please use our ``line-height()`` mixin by passing in the size of the font you're rendering - the mixin calculates a 1.48 scale value for it (e.g. (``@include line-height(18)``)).